pods-framework/pods
# Fix z-index issue with WP editor

## Related GitHub issue(s)




Fixes #6452

## Testing instructions



- Create/extend Pod
- Add relationship field (list view) and wysiwyg field (after relationship)
- Make sure there are several relationship options
- The select box is under the editor when opening. When applying this patch it's visible!

## Changelog text for these changes







## PR checklist
